plugins/CoreUpdater/Controller.php

```diff
@@ -175,11 +175,19 @@ public function oneClickUpdate()
 
     public function oneClickUpdatePartTwo()
     {
-        Piwik::checkUserHasSuperUserAccess();
```

Comment:
This shouldn't be removed?

Reply:
Oh, nvm I see it below.

Reply:
moved it to the try catch, so the error will be returned as json